pages/content/ch3/functions.mdx

```diff
@@ -13,9 +13,11 @@ For some function $f: A \to B$, the image of $A' \subseteq A$ are all elements t
 
 ## Preimage
 
-The preimage of $B' \subset B$ under $f: A \to B$ are all elements in $A$ that map to some element in $B'$, i.e. $f^{-1}(B') = \{ a \in A \; \vert \; f(a) \in B' \}$.
+The preimage of $B' \subseteq B$ under $f: A \to B$ are all elements in $A$ that map to some element in $B'$, i.e. $f^{-1}(B') = \{ a \in A \; \vert \; f(a) \in B' \}$.
 
-While the inverse $f^{-1}$ might not be defined, the preimage is always defined.
+<Callout type="warning" emoji="💡">
+    While the inverse $f^{-1}$ might not be defined, the preimage is always defined.
+</Callout>
```

Comment:
Maybe add somewhere (outside the callout) that a function is invertible if its preimage relation is a function.

Reply:
it belongs to the callout though ...